MigrationToNextDocs/Tests/odataAdapter.test.ts

```
@@ -0,0 +1,335 @@
+
+  _generateFunctionUrl(functionName: string, params: any, url?: string): string {
+    const resultUrl = `${url || 'http://localhost'}/${functionName}(`;
+    let i = 0;
+    let counter = 0;
+    for (const key in params) {
+      counter++;
+    }
+    for (const key in params) {
+      i++;
+      if (i < counter) {
+        resultUrl += `${key}='${params[key]}',`;
```

Biome (2.4.4)

[error] Can't assign resultUrl because it's a constant.

(lint/correctness/noConstAssign)